.idea/master.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="PYTHON_MODULE" version="4">
```

Comment:
These `.idea` files are from Pycharm I believe. Can you `git rm --cached` these files to get them out of the repository.